cmd/kubelet/app/server.go

```diff
@@ -1063,6 +1064,7 @@ func startKubelet(k kubelet.Bootstrap, podCfg *config.PodConfig, kubeCfg *kubele
 	}
 }
 
+//CreateAndInitKubelet - creates and instantiate kubelet
```

Comment:
```suggestion
//CreateAndInitKubelet creates and instantiate kubelet
```

Reply:
@resouer 
Requested change is implemented.